nemo/core/config/__init__.py

```diff
@@ -15,7 +15,31 @@
 # =============================================================================
 
 from nemo.core.config.base_config import Config
-from nemo.core.config.optimizers import AdamConfig, AdamParams, NovogradConfig, NovogradParams
+from nemo.core.config.optimizers import (
+    AdadeltaParams,
+    AdagradParams,
+    AdamaxParams,
+    AdamParams,
+    AdamWParams,
+    NovogradParams,
+    RMSpropParams,
+    RpropParams,
+    SGDParams,
+    get_optimizer_config,
+)
 from nemo.core.config.pytorch import DataLoaderConfig
 from nemo.core.config.pytorch_lightning import TrainerConfig
+from nemo.core.config.schedulers import (
+    CosineAnnealingParams,
```

Comment:
Question @okuchaiev Do we want to support pytorch standard schedulers ? We'll need to create configs for them then.

Reply:
I think we should.

Reply:
So I cant (yet) support LambdaLR, MultiplicativeLR, MultiStepLR, due to it needing a function as an argument, and I cant instantiate it functions from config dict. Im dropping PT `CosineAnnealingLR` cause we have our own (imo better) CosineAnnealing scheduler.

Reply:
let's skip them for now